storage/integration_test.go

```diff
@@ -744,6 +746,127 @@ func TestIntegration_ObjectsRangeReader(t *testing.T) {
 	}
 }
 
+func TestIntegration_ObjectReadGRPC(t *testing.T) {
+	// TODO: Remove this once v2 rollout is complete.
+	t.SkipNow()
+
+	ctx := context.Background()
+	client, err := newClientWithGRPC(ctx)
+	if err != nil {
+		t.Error(err)
+	}
+	defer client.Close()
+	bkt := client.Bucket(grpcBucket)
+
+	objName := uidSpace.New()
+	obj := bkt.Object(objName)
+	contents := []byte("Hello, world this is a grpc request")
+
+	if err := retry(ctx, func() error {
```

Comment:
Should be no need to retry with current write logic since the writer itself retries.

Reply:
Thanks, I just copied the code from another test, but I'll remove that stuff here.

Reply:
Removed and refactored into a separate function that creates a new HTTP client. Done!